libsolidity/analysis/StaticAnalyzer.cpp

```diff
@@ -68,8 +68,17 @@ void StaticAnalyzer::endVisit(FunctionDefinition const&)
 	m_nonPayablePublic = false;
 	m_constructor = false;
 	for (auto const& var: m_localVarUseCount)
-		if (var.second == 0)
-			m_errorReporter.warning(var.first->location(), "Unused local variable");
+		if ( var.second == 0 ) {
+			if(var.first->isCallableParameter())
+				m_errorReporter.warning(
+					var.first->location(), 
+				"Unused function parameter. Remove or comment out the variable name to silence this warning"
```

Comment:
This line should be indented one tab further. Also please add a full stop to the end of both strings.